examples/bri-3/components/identity/bpiAccount/bpiAccount.ts

```diff
@@ -0,0 +1,12 @@
+import { IBpiAccount } from "./bpiAccount.interface";
+
+export class BpiAccount implements IBpiAccount {
+    id: string; // TODO: Add uuid after #491
+    nonce: number;
```

Comment:
since there is more than one way to generate a deterministic nonce a string might be more appropriate

Reply:
Considering the upper limit of TS number type, I think we shall implement it as bignumber right since the beginning.